Updated the installation doc (#3270)
wso2/docs-is: en/docs/deploy/get-started/install.md

<th><p>Operating Systems/ Databases/ Userstores</p></th>
<td><div class="content-wrapper">
<ul>

> <li>WSO2 Identity Server requires an Oracle JDK 17 or JDK 11 compliant JDK. This will run on most common platforms that <strong>support Java 17 or Java 11.</strong> .</li>

Reviewer: ```suggestion
<li>WSO2 Identity Server requires a JDK 11 or JDK 17 compliant JDK. This will run on most common platforms that <strong>support Java 11 or Java 17.</strong> .</li>
```
